build.sbt

```diff
@@ -18,11 +18,17 @@ ThisBuild / githubWorkflowBuild := Seq(
     name = Some("Build native assembly")
   )
 )
-ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.8", "graalvm-ce-java11@")
-ThisBuild / githubWorkflowOSes := Seq("ubuntu-latest", "macos-latest")
-ThisBuild / githubWorkflowBuildMatrixExclusions := Seq(MatrixExclude(Map("os" -> "macos-latest", "java" -> "adopt@1.8")))
+
+val PrimaryOS = "ubuntu-latest"
+ThisBuild / githubWorkflowOSes := Seq(PrimaryOS)
```

Comment:
I'm not convinced if it's super important, but could we preserve `macos` too?

Reply:
Yeah, I thought it wouldn't bring much value tbh, but I'll try to restore that.